mailbbgun/views.py

```diff
@@ -0,0 +1,189 @@
+import flask
+import jsonschema
+import logging
+import pika
+
+from mailbbgun import app
+from mailbbgun import models
+
+
+if app.config['DEBUG']:
+    logging.basicConfig(level=logging.DEBUG)
+
+
+_DELIVERY_MODE_PERSISTENT = 2
+_LOG = logging.getLogger(__name__)
+
+
+class BadRequestException(Exception):
+    def __init__(self, message):
+        super().__init__(message)
+        self.message = message
+
+
+@app.route('/messages', methods=['POST'])
+def new_message():
+    data = _validate_message_request()
+    _LOG.debug("New message: {}".format(data))
+
+    message = models.Message(**data)
+    models.db.session.add(message)
+
+    connection = pika.BlockingConnection(
```

Comment:
I'm assuming this isn't creating a new connection for each api call and `pika` pools connections behind the scenes